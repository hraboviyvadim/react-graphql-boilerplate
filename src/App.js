import React from 'react';

const App = () => {
  return <h1>Hello, world2!</h1>;
};

export default App;
