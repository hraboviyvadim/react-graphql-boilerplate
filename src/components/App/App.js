import React from 'react';

const App = () => {
  return <h1>Hello, world3!</h1>;
};

export default App;
