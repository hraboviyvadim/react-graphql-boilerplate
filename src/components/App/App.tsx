import React from 'react';

interface AppProps {
  name: String;
}

const App = (props: AppProps) => {
  return <h1>Hello, {props.name}!</h1>;
};

export default App;
