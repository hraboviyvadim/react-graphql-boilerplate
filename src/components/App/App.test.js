import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

describe('Unit test example', () => {
  it('My Test Case', () => {
    expect(true).toEqual(true);
  });
});

describe('Snapshot test example', () => {
  it('Shallow render', () => {
    const shallowed = shallow(<App name="world3" />);
    expect(shallowed).toMatchSnapshot();
  });
});
