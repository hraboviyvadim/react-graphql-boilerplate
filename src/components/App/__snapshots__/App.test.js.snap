// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test example Shallow render 1`] = `
<h1>
  Hello, 
  world3
  !
</h1>
`;
